src/CommonLib/Helpers.cs

```diff
@@ -327,6 +327,11 @@ public static IRegistryKey OpenRemoteRegistry(string target)
             CommonOids.SmartcardLogon,
             CommonOids.AnyPurpose
         };
+
+        public static bool IsSid(string sidContender)
+        {
+            return sidContender != null && sidContender.StartsWith("S-1-");
```

Comment:
This isn't really that useful anymore.  The logic really isn't that prolific, and the places it's currently used can be replaced with simple 'not null' checks.